README.md

```diff
@@ -1,32 +1,21 @@
 # gpu-info
-A useful library that provides GPU info (it's in the name!)
 
-The purpose of this library is to fetch GPU information that are not accessible through pure JS.
+This library fetches Windows GPU information not accessible through pure JavaScript.
 
-It is not intended to build/run this library on any other platform that Windows. 
-The library is build and integrated to Node.js with [node-gyp](https://github.com/nodejs/node-gyp).
-
-## Development
-
-Windows users need to have the msvc (Visual Studio 2017) build tools set up on their system.
-
-```shell
-npm i -g node-gyp
-yarn add node-addon-api
-npm i -g windows-build-tools --vs2017
-npm config set msvs_version 2017
-
-# Driver kit Import path in 'C:/Program Files (x86)/Windows Kits/10/Include/10.0.17763.0/km'
-choco install windowsdriverkit10 --version 10.0.17763
-```
+- [Usage](#usage)
+- [API](#api)
+  - [API interface](#api-interface)
+  - [Implemented data points](#implemented-data-points)
+- [Development](#development)
+  - [Prerequisites](#prerequisites)
 
 ## Usage
 
-The C++ code can be compiled into the `<package>/build/Release/gpu-metric.node` file.
-This file can be imported through:
+This library is only supported on 64-bit versions of Windows 10.
 
 ```js
-const addon = require('<path to>/build/Release/gpu-info.node');
+const windowsGpuInfo = require('@saladtech/gpu-info');
```

Comment:
Is this an alias? GitHub suggests it as `@saladtechnologies/gpu-info`

Reply:
GitHub Packages requires the `@scope` to match the GitHub organization name. Right now these are mismatched, as our scope/username on npm is "saladtech". I dropped a card into the backlog to fix the inconsistency.